content/plugins/filament-spatie-roles-permissions.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Spatie Roles Permissions
+slug: filament-spatie-roles-permissions
```

Comment:
This should be `tharinda-rodrigo-spatie-roles-permissions`, same as the file name